mac/build.sh

```diff
@@ -263,7 +263,7 @@ do_sentry() {
 }
 
 do_install() {
-  if ! builder_has_option --quick; then
+  if builder_is_ci_build_level_release && ! builder_has_option --quick; then
```

Comment:
I'm not exactly up on all the latest variants of this, but it doesn't look like `builder_is_ci_build_level_release` covers both standard releases _and_ local builds.  Wouldn't notarization be needed for a local install / build iteration during development?

Alternatively... does this need a `builder_is_ci_build` check before using `builder_is_ci_build_level_release`?

https://github.com/keymanapp/keyman/blob/40795d6ab7ef55e6dc8d872a057aeaf6332a7ce7/resources/builder.inc.sh#L2228-L2232

Reply:
Yeah, I got this wrong here (conflating `builder_if_release_build_level` and `builder_is_ci_build_level_release`). `install` should only be run on local anyway, so we should die if it happens in CI. Then we don't need to change the previous test.